spec/omniauth/strategies/saml_spec.rb

```diff
@@ -222,4 +222,13 @@ def post_xml(xml=:example_response)
   it 'implements #on_metadata_path?' do
     expect(described_class.new(nil)).to respond_to(:on_metadata_path?)
   end
+
+  describe 'subclasses' do
+    let(:subclass) { Class.new(described_class) }
```

Comment:
instead of declaring the `subclass` in a `let` can we just initialize it within the `it` block?

```ruby
it 'registers subclasses in OmniAuth.strategies' do
  subclass = Class.new(described_class) # or described_class.new
  expect(OmniAuth.strategies).to include(subclass)
end
```

I find this approach easier to read as it defines everything the test needs within the test itself.